### DISCUSSION TITLE
I can’t verify Academic status

### DISCUSSION THREAD

**Discussion by ghost** [community/community#69535] [GitHub Education]
### Hi everyone,

every time I trying to proof that i’m a student the error keeps showing up:
“Your application cannot be reviewed until you fix the following:
 • Please correct your GitHub billing information with your last name exactly as it appears in your academic affiliation document.
 • Please correct your GitHub billing information with your first name exactly as it appears in your academic affiliation document. Do not use a variation of your name or a nickname.”
And I did all correct I don’t know what I should do

**Comment by mantasu** [comment]
Hi, I noticed in other discussions that people have some translation issues, [this answer](https://github.com/orgs/community/discussions/68725#discussioncomment-7163466) could help. And also you can always open a [ticket](https://support.github.com/request/education), which is what I did when I was facing issues with student benefits.

**Comment by santoshvandari** [comment]


**Reply by Zeber0** [reply]
Got same problem. But for now you cant send a ticket to a support about verification problems. Is there any other options to get a non automatic human help?

**Reply by liya-oz** [reply]
@Zeber0 Hi! Could you solve the problem?
